Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -34,6 +34,7 @@ enum BuiltInModules
             Shaders = 1 << 2,
             Resources = 1 << 3,
             BuildReport = 1 << 4,
+            InstalledPackages = 1 << 5,
```

Comment:
since this module is currently "hidden" as developer/experimental and the module selection is being refactored, please remove from the Home page